packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/About.1.tool.mdx

```diff
@@ -445,3 +445,13 @@ it('should update an application's locale, () => {
   Terra.setApplicationLocale('en');
 });
 ```
+
+## Remote Screenshots Migration
+
+<Notice variant="important" ariaLevel="3">
+
+  The Remote Screenshots Mirgation feature is currently only supported by internal projects built using Jenkins at Cerner Corporation. Open source projects that utilize other CI such as Travis are not supported.
+
+</Notice>
+
+Follow this [consumption guide](https://pages.github.cerner.com/orion/docs/doc/docs/nexus-wdio-screenshots-mirgation/about) to utilize this feature.
```

Comment:
@BenBoersma suggested to limit internal references in terra-ui site for this guide, which Jaime and I agreed. For this reason, I moved the guide to an internal repo and link it here.